.github/workflows/quality-gates.yaml

```
@@ -27,3 +27,9 @@ jobs:
     secrets: inherit
     with:
       charm-path: backend
+  terraform-tests:
+    name: Run terraform tests
+    uses: ./.github/workflows/_terraform-tests-auto.yaml
```

I was thinking we could use the one in otel-ebpf-profiler instead of copy-pasting it here.
Or, first put it in observability?
```suggestion
  # something like:
    uses: github.com/canonical/otel-ebpf-profiler-operator/blob/main/.github/workflows/_terraform-tests-auto.yaml
```

---

BTW, this one and the one in otel are not identical, so the reference wouldn't work.

---

ah ok. I thought the one I put on otel was already generic enough. Sure then, let's go ahead.